jenkins/e2e-image/e2e-runner.sh

```diff
@@ -369,9 +369,15 @@ fi
 
 cd kubernetes
 
-# Upload build start time and k8s version to GCS, but not on PR Jenkins.
-# On PR Jenkins this is done before the build.
-if [[ ! "${JOB_NAME}" =~ -pull- ]]; then
+if [[ -n "${BOOTLOADER_MIGRATION:-}" ]]; then
+  # TODO(fejta): migrate all jobs and stop using upload-to-gcs.sh to do this
+  source "$(dirname "${0}")/upload-to-gcs.sh"
+  print_started | jq '.metadata?' > "${ARTIFACTS}/metadata.json"
+elif [[ ! "${JOB_NAME}" =~ -pull- ]]; then
+  echo "The bootstrapper should handle Tracking the start/finish of a job and "
```

Comment:
is this message helpful to people reading build logs? it mostly is just confusing if you haven't seen this PR.